bsconfig.json

```diff
@@ -0,0 +1,13 @@
+/* bsconfig.json */
+{
+  "name": "wp-calypso",
+  "sources": [
+    "client/bs"
```

Comment:
I'm not sure this works as intended? Schema at http://bloomberg.github.io/bucklescript/docson/#build-schema.json

`[{dir: "client", subdirs: "bs"}]`